### ISSUE TITLE
Activation: 'LeakyReLU'

### ISSUE THREAD

**Issue by zinuok** [philipperemy/keras-tcn#217]
Is there a separate list of activation functions available? I searched [model definition](https://github.com/philipperemy/keras-tcn/blob/master/tcn/tcn.py), but only [linear, sigmoid, and relu] are used, so I wonder if only these three are supported.

I want to use LeakyReLU, but is it a feature that is not supported?

Beste regards,
Jinwoo Jeon

**Comment by philipperemy** [comment]
@zinuok hey sorry for the late reply. I'm going to push an update for you to be able to specify other activation functions like LeaKyRelu.

**Comment by philipperemy** [comment]
@zinuok sorry for the late reply.

Yes (almost) all activations here are supported: 

![image](https://user-images.githubusercontent.com/4516927/150288061-32ed3284-b55a-4ce7-8155-6344ecb06a5b.png)
From https://keras.io/api/layers/


**Example**

```python
from tensorflow.keras import Sequential
from tensorflow.keras.layers import Dense, Dropout, Embedding

from tcn import TCN

model = Sequential()
model.add(Embedding(2000, 128, input_shape=(10,)))
model.add(TCN(
    kernel_size=6,
    activation='LeakyReLU',          # <---------------- leaky relu, elu...
    dilations=[1, 2, 4, 8, 16, 32, 64]
))
model.add(Dropout(0.5))
model.add(Dense(1, activation='sigmoid'))

model.summary()

```
